tools/protoc-gen-mypy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# use PYTHON_EXECUTABLE from CMake to get the right python to execute
+PYTHON_EXECUTABLE=$(grep PYTHON_EXECUTABLE:FILEPATH CMakeCache.txt | cut -d "=" -f2)
```

Comment:
See https://cmake.org/cmake/help/v3.19/command/configure_file.html

Reply:
Thank you for this. Updated. It is more elegant now.